src/Misc/layoutbin/installdependencies.sh

```diff
@@ -204,7 +204,7 @@ then
                 fi
 
                 # install lttng-ust separately since it's not part of offical package repository
-                yum install -y wget && wget -P /etc/yum.repos.d/ https://packages.efficios.com/repo.files/EfficiOS-RHEL7-x86-64.repo && rpmkeys --import https://packages.efficios.com/rhel/repo.key && yum updateinfo -y && yum install -y lttng-ust
+                yum install -y https://cbs.centos.org/kojifiles/packages/userspace-rcu/0.10.0/3.el7/x86_64/userspace-rcu-0.10.0-3.el7.x86_64.rpm https://cbs.centos.org/kojifiles/packages/lttng-ust/2.10.0/1.el7/x86_64/lttng-ust-2.10.0-1.el7.x86_64.rpm
```

Comment:
Let's check if it's possible to download it without specifying specific package version.